constants.py

```diff
@@ -34,7 +34,7 @@
                     "3": "captures/buttons/tab_tft_unselected_3.png",
                     "4": "captures/buttons/tab_tft_unselected_highlighted.png",
                 },
-                "subtab_home": "captures/buttons/tft_subtab_home.png"
+                "subtab_home": "captures/buttons/tft_subtab_home.png",
```

Comment:
Another one that may be a bit controversial -- I'm a bit fan of trailing commas in multi-line arrays/objects (though for single line I find that cleaner not to). Looks like [PEP-8 also makes this optional](https://peps.python.org/pep-0008/#when-to-use-trailing-commas).

An example of why I prefer this is [this tidbit](https://stackoverflow.com/a/11597911/4654928), where you essentially skip having "fake" extra lines in commit diffs, in addition to the readability.

I do feel a bit stronger on this one, but again not a blocker to me since this was default provided by one of the formatters.

Reply:
I may be mis-reading this, but this is **adding** a trailing comma, which you're a fan of? So it's a good thing, no?

Reply:
.. 🤦‍♂️